src/js/options.js

```diff
@@ -953,6 +953,16 @@ function activateDomainListTooltips() {
   $rows.find('.honeybadgerPowered.tooltip').tooltipster();
 }
 
+$(document).on('keydown', function(event) {
```

Comment:
Instead of listening globally in the entire document, let's add a keydown listener to the individual domain summaries.